.github/workflows/ci.yml

```diff
@@ -37,12 +43,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: dtolnay/rust-toolchain@nightly
-        with:
-          components: miri
+      - uses: dtolnay/rust-toolchain@miri
       - run: cargo miri test
         env:
-          MIRIFLAGS: "-Zmiri-tag-raw-pointers"
+          MIRIFLAGS: -Zmiri-strict-provenance
```

Comment:
I like that they are using miri here, maybe we could also use it in Boa.

Reply:
Yes, I think so too maybe it could catch some hard to find bugs

Reply:
Let's open an issue about this